### ISSUE TITLE
Octo-STS as bypasser for Rule sets

### ISSUE THREAD

**Issue by shyim** [octo-sts/app#594]
Hey,

We have a Ruleset and I wanna bypass PR's made by Octo-STS. Therefore I selected the Octo-STS app as an bypasser, but this does not have an effect and the ruleset is still active.

Do you have experience with that?

**Comment by mattmoor** [comment]
Yeah, we do this to allow bypassing some of our org-level rulesets.  Here's one that allows our automation to create branches on the upstream (instead of needing a fork): 
<img width="956" alt="image" src="https://github.com/user-attachments/assets/04ffd139-16d1-418c-8d1b-76214568a8fd">
